src/utils/discord/logger.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+
+const padZeros = (value, length = 2) => {
```

Comment:
you can reuse zeroPad from https://github.com/elmadev/elmaonline-site/blob/dev/src/utils/time.js